### ISSUE TITLE
Add partial alignment support to MakeDb

### ISSUE THREAD

**Issue by ssnn-airr** [immcantation/changeo#80]
**[Original report](https://bitbucket.org/kleinstein/changeo/issues/80) by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

We should add support for partial alignments, such as V only and CDR3+J only to the MakeDb output. @ssnn-airr partially completed this in ba61750.

My inclination would be to add a `--partial` flag to include partial alignments in the pass file, with the default behavior being to put them in the fail file with all available parsed information (as per ba61750). That way, it's explicit as to whether or not you retain partial alignments.

The behavior needs to be consistent across the IMGT and IgBLAST parsers, and will probably require a lot of testing, including:

* Compatibility with downstream steps (DefineClones, CreateGermlines)
* Verify that fields are empty/informative when they should be (eg, no `J_SEQ_START` if there is no J alignment)

**Comment by ssnn-airr** [comment]
**Original comment by ssnn (Bitbucket: [ssnn](https://bitbucket.org/ssnn), GitHub: [ssnn-airr](https://github.com/ssnn-airr)).**

----------------------------------------

2642661 Needs testing

**Comment by ssnn-airr** [comment]
**Original comment by Jason Vander Heiden (Bitbucket: [javh](https://bitbucket.org/javh), GitHub: [javh](https://github.com/javh)).**

----------------------------------------

I fixed where the `--partial` flag defined in d5b514c (arguments should only go into `changeo.Commandline.getCommonArgParser` if a majority of tools have the argument). Tested the various commands and no hiccups. Not a thorough test, but we need to full test of MakeDb because of all the recent changes, so I'm closing this.
